### ISSUE TITLE
React 16 (peer dependencies)

### ISSUE THREAD

**Issue by NehrDani** [twobucks/react-gcaptcha#7]
It seems to have trouble with react 16.
At least the peer dependencies needs an update.

Here the `npm` warning:
```
npm WARN react-gcaptcha@3.10.0 requires a peer of react@15.x but none is installed. You must install peer dependencies yourself.
```

**Comment by acidicX** [comment]
@dinodsaurus that one would be really nice :-)

**Comment by dinodsaurus** [comment]
I will do it this week but yea I accept pull requests :)

**Comment by dinodsaurus** [comment]
Ok, should be fixed on `4.0.0`
